cmd/kubeadm/app/node/csr.go

```diff
@@ -18,16 +18,18 @@ package node
 
 import (
 	"fmt"
+	"os"
 
 	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
+	"k8s.io/kubernetes/pkg/client/unversioned/clientcmd"
 	clientcmdapi "k8s.io/kubernetes/pkg/client/unversioned/clientcmd/api"
 	"k8s.io/kubernetes/pkg/kubelet/util/csr"
+	"k8s.io/kubernetes/pkg/types"
 	certutil "k8s.io/kubernetes/pkg/util/cert"
 )
 
-// PerformTLSBootstrap executes a certificate signing request with the
-// provided connection details.
-func PerformTLSBootstrap(connection *ConnectionDetails) (*clientcmdapi.Config, error) {
+func PerformTLSBootstrapDeprecated(connection *ConnectionDetails) (*clientcmdapi.Config, error) {
```

Comment:
Can't we just squash these two methods together or is that yet to be done in a follow up?
If that's the case, add a TODO